.github/workflows/ci.yml

```diff
@@ -24,8 +24,8 @@ jobs:
       - uses: software-mansion/setup-scarb@v1.0.0
         with:
           scarb-version: ${{ env.SCARB_VERSION }}
-      - run: cargo test --lib --manifest-path ./starknet-foundry/Cargo.toml -p forge
-      - run: cargo test integration --manifest-path ./starknet-foundry/Cargo.toml -p forge
+      - run: cargo test --lib --manifest-path ./Cargo.toml -p forge
+      - run: cargo test integration --manifest-path ./Cargo.toml -p forge
```

Comment:
Not sure, but we may not need to pass `--manifest-path` now if the `Cargo.toml` is in the top-level anyways

Reply:
Same goes for code below, though it requires veryfing